### PR TITLE
LPS-179361 [PRM] Allow Deals fragment to be instantiable 

### PR DESCRIPTION
**Pull request by claricebrasil**

### CODE REVIEW

**Comment by claricebrasil** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layouts/04_sales/02-partner-opportunities/page-definition.json [hunk 1]
```diff
@@ -21,14 +25,96 @@
...
+																}
+															]
+														}
+													}
+												}
+											}
+										},
+										"text": {
+											"value_i18n": {
+												"en_US": "Partner Opportunities"
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layouts/04_sales/02-partner-opportunities/page-definition.json:96]: Add another languages

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layouts/04_sales/02-partner-opportunities/page-definition.json [hunk 2]
```diff
@@ -21,14 +25,96 @@
...
+																}
+															]
+														}
+													}
+												}
+											}
+										},
+										"text": {
+											"value_i18n": {
+												"en_US": "Partner Deal Registration"
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layouts/04_sales/02-partner-opportunities/page-definition.json:65]: Add another languages

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layouts/04_sales/01_deal-registration/page-definition.json [hunk 3]
```diff
@@ -21,14 +25,96 @@
...
+																}
+															]
+														}
+													}
+												}
+											}
+										},
+										"text": {
+											"value_i18n": {
+												"en_US": "Partner Opportunities"
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layouts/04_sales/01_deal-registration/page-definition.json:96]: Add another languages

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layouts/04_sales/01_deal-registration/page-definition.json [hunk 4]
```diff
@@ -21,14 +25,96 @@
...
+																}
+															]
+														}
+													}
+												}
+											}
+										},
+										"text": {
+											"value_i18n": {
+												"en_US": "Partner Deal Registration"
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layouts/04_sales/01_deal-registration/page-definition.json:65]: Add another languages

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layouts/03_marketing/03_mdf-claim/page-definition.json [hunk 5]
```diff
@@ -35,6 +39,110 @@
...
+																				}
+																			]
+																		}
+																	}
+																}
+															}
+														},
+														"text": {
+															"value_i18n": {
+																"en_US": "MDF Claim"
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layouts/03_marketing/03_mdf-claim/page-definition.json:118]: Add another languages

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layouts/03_marketing/03_mdf-claim/page-definition.json [hunk 6]
```diff
@@ -35,6 +39,110 @@
...
+																				}
+																			]
+																		}
+																	}
+																}
+															}
+														},
+														"text": {
+															"value_i18n": {
+																"en_US": "MDF Request"
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layouts/03_marketing/03_mdf-claim/page-definition.json:87]: Add another languages

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layouts/03_marketing/01_mdf-requests/page-definition.json [hunk 7]
```diff
@@ -35,6 +39,110 @@
...
+																				}
+																			]
+																		}
+																	}
+																}
+															}
+														},
+														"text": {
+															"value_i18n": {
+																"en_US": "MDF Claim"
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layouts/03_marketing/01_mdf-requests/page-definition.json:118]: Add another languages

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layouts/03_marketing/01_mdf-requests/page-definition.json [hunk 8]
```diff
@@ -35,6 +39,110 @@
...
+																				}
+																			]
+																		}
+																	}
+																}
+															}
+														},
+														"text": {
+															"value_i18n": {
+																"en_US": "MDF Request"
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layouts/03_marketing/01_mdf-requests/page-definition.json:87]: Add another languages

**Review by jairmedeiros — CHANGES_REQUESTED** [review]

**Comment by claricebrasil** [comment]
ci:test:sf

**Comment by claricebrasil** [comment]
ci:forward

**Pull request closed without merging**